packages/openapi-generator/src/cli/index.ts

```diff
@@ -110,13 +113,24 @@ $ openapi-generator --input ./my-spec.yaml --outputDir ./client --transpile`
       description: 'When enabled, metadata for the API hub is generated.',
       default: false,
       hidden: true
+    }),
+    config: flags.string({
+      char: 'c',
+      parse: input => resolve(input),
+      description: 'Set the path to the config.json file for generation.',
+      helpValue: '<path/to/config.json>',
+      required: false
     })
   };
 
   async run(): Promise<void> {
     try {
       const parsed = this.parse(OpenApiGenerator);
-      await generateWithParsedOptions(parsed.flags);
+      if (parsed.flags.config) {
+        await generate(parseOptionsFromConfig(parsed.flags.config));
+      } else {
+        await generateWithParsedOptions(parsed.flags);
+      }
```

Comment:
This means we cannot use both config file and command line options, which is fine.
However, this is possible with the OData generator (not sure, who wins when reaching conflicts though).
We should make a decision for the alignment and make it explicit in the docs.

Personally, I'm not a big fan of using the `parse` (line 128) provided by the lib and using `parseOptionsFromConfig` (line 130) made by us together, which might be error prone.

Reply:
Agreed, I've implemented it (with some additions) and now additional flags are overwriting the options that are set in the config file.